src/octopeer-github/main/Database/ConsoleLogDatabaseAdapter.ts

```diff
@@ -1,12 +1,32 @@
 /**
  * Created by Youri on 04/05/2016.
- */
-/**
  * Semi-mock class for being able to check implementation completeness.
  */
 class ConsoleLogDatabaseAdapter implements DatabaseAdaptable {
 
     /**
+     * Simple boolean to keep track of the raw data logging.
+     */
+    private static rawDataLogging = true;
+
+    /**
+     * Set the raw data switch for the console.
+     * By default the console will show raw data.
+     * @logging the new value.
+     */
+    public static setRawData(logging = true) {
```

Comment:
I would prefer it if the naming would be consistent with the field name, so setRawDataLogging. (same goes for the getter)